README.md

```diff
@@ -1 +1,175 @@
-# tomcat-extension-openidsso
\ No newline at end of file
+# Tomcat Extension - OpenID Connect based Single Sign On and Single Logout #
+
+ OpenID Connect (OIDC) is an identity layer protocol on top of OAuth 2.0.
+ This extension provides the capability of enabling SSO and SLO using OIDC for user applications via WSO2 Identity Server.
+
+ Follow the below steps to see how this extension works.
+
+ We will use two web applications named ‘pizza-shop’ and ‘coffee-shop’  with WSO2 Identity server.
+
+## Step 1: Download and install Tomcat 8 and WSO2 IS ##
+
+ Tomcat 8 will be used to deploy web applications and WSO2 IS will be used as the identity provider that enables SSO and SLO.
+
+1. Download Tomcat 8 and extract the zip file to your computer. 
+   The extracted directory will be your &lt;Tomcat_HOME&gt; directory.
+2. Download WSO2 IS and extract the zip file to your computer. 
+   The extracted directory will be your &lt;IS_HOME&gt; directory.
+
```

Comment:
Shall we provide the download links to Apache Tomcat and WSO2 IS, if there is a specific IS version to be used, we better mention that as well.